tests/pipeline/test_numerical_expression.py

```diff
@@ -147,6 +147,41 @@ def test_validate_bad(self):
         with self.assertRaises(TypeError):
             (f > f) > f
 
+    def test_many_inputs(self):
+        factors = []
+
+        # Create 11 factors to use as inputs. We need at least 11 to create a
+        # string expression containing an x_10. Then, ensure that this long
+        # expression can correctly merge with another.
+        for class_name in 'PQRSTUVWXYZ':
```

Comment:
Might be worth going up a bit higher here:

```
from string import ascii_uppercase
[a + b for a, b in product(ascii_uppercase, ascii_uppercase)]
```
^ will give you all length-two pairs of uppercase letters